app/assets/v2/js/shared.js

```diff
@@ -1000,48 +1000,6 @@ window.addEventListener('load', function() {
   setInterval(listen_for_web3_changes, 1000);
 });
 
-var callMethodIfTokenIsAuthed = function(success, failure) {
-  var denomination = $('#token option:selected').text();
-  var tokenAddress = $('#token option:selected').val();
-
-  if (!denomination) {
-    failure(denomination, tokenAddress);
-  } else if (denomination == 'ETH') {
-    success(denomination, tokenAddress);
-  } else {
-    var token_contract = web3.eth.contract(token_abi).at(tokenAddress);
-    var from = web3.eth.coinbase;
-    var to = bounty_address();
-
-    token_contract.allowance.call(from, to, function(error, result) {
-      if (error || result.toNumber() == 0) {
-        failure(denomination, tokenAddress);
-      } else {
-        success(denomination, tokenAddress);
-      }
-    });
-  }
-};
-
-var promptForAuthFailure = function(denomination, tokenAddress) {
-  _alert(
-    gettext(`To enable this token, go to the
-    <a style="padding-left:5px;" href="/settings/tokens">
-    Token Settings page and enable it.
-    </a> This is only needed once per token.`),
-    'warning'
-  );
-};
-
-var promptForAuth = function(event) {
-
-  var success = function(denomination, tokenAddress) {
-    $('.alert').remove();
-  };
-
-  callMethodIfTokenIsAuthed(success, promptForAuthFailure);
-};
```

Comment:
moved the above into new_bounty.js and refactored to make it into 2 neat functions